scripts/utils.py

```diff
@@ -0,0 +1,123 @@
+from binascii import unhexlify, hexlify
+import base64
+import re
+import argparse
+
+def Base64ToHexHash(base64_hash):
+    return hexlify(base64.decodestring(base64_hash.strip('\n"\'')))
+
+#st = Base64ToHexHash('YXg/2Z/PYhMRgRXqxJBjpg==')
```

Comment:
Whats this?

Reply:
I have just removed that line at 77f150cb19e61bffaf580ba1b5531739f02920ae